README.md

```diff
@@ -87,6 +87,10 @@ Initialize the Datadog tracer when set to `true`. Defaults to `false`.
 
 Set to `true` to merge the X-Ray trace and the Datadog trace, when using both the X-Ray and Datadog tracing. Defaults to `false`.
 
+### DD_FLUSH_IN_THREAD
+
+Flush metrics periodically in a background thread. See details in this [PR](https://github.com/DataDog/datadog-lambda-python/pull/120). Defaults to `false`.
+
```

Comment:
I wouldn't include a link to a PR in the README, I would just explain directly in the README.

With that said, why do we have this as an environment variable? In the PR description, you wrote "In short, the background thread flushing isn't suitable for Lambda and should be disabled." If it's not suitable/should be disabled, I think exposing an option to enable it might lead to confusion with little or no benefit for customers. Maybe we should just force it to be off.

If we really want to preserve the ability to turn this environment variable on for some edge cases, I would recommend that we do one or both of the following:
- Rename the variable to `DD_LEGACY_METRICS_FLUSHING_BEHAVIOR` to discourage people from using it.
- Include the variable in the code and add a comment, but don't include it in the README to avoid confusing new users.